### ISSUE TITLE
[Screen Reader - Identity Diagnostics - Virtual Assistant (Preview)] Screen reader doesn't announce updated message information which is provided by 'Virtual assistant'.

### ISSUE THREAD

**Issue by test438** [microsoft/BotFramework-WebChat#3484]
**User Experience:**
Properly defined information & relationship of an UI element helps people with different disabilities by allowing user agents to provide proper navigation & content according to the needs of individual users.

**Note:** User credentials should NOT be included in the bug.

 

**Repro Steps:​**
1. Launch the URL: https://aka.ms/dxp-ux  and login with credentials.
2. Navigate to the Enterprise applications  and select it.
3. Navigate to the "Diagnose and solve problems
" and select .
4. Navigate to the various controls of the "Virtual assistant (Preview)
" Blade .
5. Verify that Screen reader announces updated message information which is provided by 'Virtual assistant' or not.

**Actual Result:**
Screen reader doesn't announce updated message information which is provided by 'Virtual assistant' that is 'Sorry, I didn't get that. Did you mean:....' after providing question by user.

**Expected Result:**
Screen reader should announce updated message information which is provided by 'Virtual assistant' that is 'Sorry, I didn't get that. Did you mean:....' after providing question by user.

Note:
Same issue exists on Firefox + NVDA & Chrome + JAWS.

MAS Reference:
MAS 1.3.1 Info and Relationship

![Screen reader doesn't announce updated message information which provided by 'Virtual assistant'](https://user-images.githubusercontent.com/60651748/93352131-22084800-f858-11ea-8724-7c8c98176dde.png)

**Comment by corinagum** [comment]
I verified that this is running 4.10.0 Web Chat. 
![image](https://user-images.githubusercontent.com/14900841/93398628-c8d8ee00-f830-11ea-96cb-d976f5106e0b.png)
Testing on Chrome 85.0.4 with NVDA

Here is the log from NVDA:
![image](https://user-images.githubusercontent.com/14900841/93400031-b1e7cb00-f833-11ea-9843-0bb5aec35b6e.png)

To note, the announcement flow is interrupted by a URL (not sure of the source yet), which is not good UX. But even after waiting for that to be spoken, the announcement for the Adaptive Card is "Bot VA said: 1 attachment". It should definitely be speaking the contents of the card. 

I have reproduced this issue on the Virtual Assistant, and am now investigating whether or not I can reproduce it in a separate bot. I will report back here.

**Comment by corinagum** [comment]
I'm actually confirming this bug on Mock Bot in the browser as well. Looks like we have two a11y issues that need to be resolved: https://github.com/microsoft/BotFramework-WebChat/issues/3489

**Comment by corinagum** [comment]
TLDR: I am closing this bug as "by design / won't fix"

* I am no longer reproducing the long url that is read, which interrupted the speech flow of AT

After taking a look at the source code, I believe __not reading the attachment__ is __intentional behavior__. The current ideal workflow is thus:

![image](https://user-images.githubusercontent.com/14900841/96520693-75d5ca80-1224-11eb-9f3e-676397fd99ef.png)

'Bot said, where are you located? 1 attachment'
(*Focus remains on last clicked button*)
(*User presses tab*)
Bot said, Where are you located? activity attachment (* This is the context being read)
   USA button
(*User presses tab*)
Bot said, Where are you located? activity attachment
  Other button


`ScreenReaderActivity.js`:
![image](https://user-images.githubusercontent.com/14900841/96519222-5c7f4f00-1221-11eb-8ebf-d8325f6119d3.png)

The intent here is that the context of the attachment (associated activity) will be read again when focus 'enters' the attachment.

@compulim Could I get a sanity check?
For related code changes, see https://github.com/microsoft/BotFramework-WebChat/pull/3287 (`<ScreenReaderActivity.js>`) and work item https://github.com/microsoft/BotFramework-WebChat/issues/3236

**Comment by corinagum** [comment]
Oops. It looks like this is a dupe of #3521. 

To recap: This behavior is **intended** but we want to update it in the feature-request linked above. Sorry for the confusion, folks. I'll keep this one closed in favor of the other, since that issue has a little more context.
